__tests__/app.test.js

```diff
@@ -263,3 +263,30 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  it("should delete a comment and return status 204", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(() => {
+        return db
+          .query("select * from comments where comment_id = 1")
+          .then(({ rows }) => {
+            expect(rows.length).toBe(0);
+          });
+      });
+  });
```

Comment:
Nice, and the query here is a nice touch to ensure the data has been deleted from the database